base/sysinfo.jl

```diff
@@ -98,7 +98,33 @@ Standard word size on the current machine, in bits.
 """
 const WORD_SIZE = Core.sizeof(Int) * 8
 
-global SC_CLK_TCK::Clong, CPU_NAME::String, JIT::String
+global SC_CLK_TCK::Clong
+
+"""
+    Sys.CPU_NAME::String
+
+A string representing the name of CPU.
+
+# Examples
+```
+julia> Sys.CPU_NAME
+"tigerlake"
+```
+"""
+global CPU_NAME::String
+
+"""
+    Sys.JIT::String
+
+A string representing the specific Just-In-Time (JIT) compiler being utilized in the current runtime.
+
+# Examples
+```
+julia> Sys.JIT
+"ORCJIT"
+```
```

Comment:
Needs some explanation, since the name "ORCJIT" is pretty obscure.  Also, we should add a `jldoctest` qualifier so that we are reminded to change this string if it ever changes:
````suggestion
Currently, this returns `"ORCJIT"` for the LLVM "ORC" ("On-Request Compilation") JIT library:
```jldoctest
````

Reply:
Note that you can click the "commit suggestion" button to incorporate this patch automatically.